### ISSUE TITLE
corner_radius Unknown/invalid command

### ISSUE THREAD

**Issue by hrqmonteiro** [WillPower3309/swayfx#350]
### Please fill out the following:
- **Swayfx Version:**
  - `sway -v` sway version 1.9

- **Debug Log:**
  - Run `sway -d 2> ~/sway.log` from a TTY and upload it to a pastebin, such as gist.github.com.
  - This will record information about sway's activity. Please try to keep the reproduction as brief as possible and exit sway.
  - Attach the **full** file, do not truncate it.

- **Configuration File:**
https://pastebin.com/zNg752g9

- **Stack Trace:**
  - This is only needed if sway crashes.
  - If you use systemd, you should be able to open the coredump of the most recent crash with gdb with
    `coredumpctl gdb sway` and then `bt full` to obtain the stack trace.
  - If the lines mentioning sway or wlroots have `??` for the location, your binaries were built without debug symbols. Please compile both sway and wlroots from source and try to reproduce.

- **Description:**
  - The steps you took in plain English to reproduce the problem.
  
  I am trying to set `corner_radius` and got this error in return: `corner_radius 20: Unknown/invalid command ' corner_radius'

**Comment by WillPower3309** [comment]
Looks like you have sway installed, the `-v` should output
```
swayfx version 0.4 (based on sway 1.9.0)
```

**Comment by hrqmonteiro** [comment]
> Looks like you have sway installed, the `-v` should output
> 
> ```
> swayfx version 0.4 (based on sway 1.9.0)
> ```

I don't have sway installed, only SwayFX, and i am on it loaded right now:

![a](https://github.com/user-attachments/assets/e6753fe1-cd6e-4df7-a6e7-cdffbb226fd9)

**Comment by DovieW** [comment]
```
❯ sway -v
swayfx version 0.4-5ee4d737 (Oct 30 2024, branch 'master') (based on sway 1.9.0)
```

I'm having the same issue.

**Comment by hrqmonteiro** [comment]
> ```
> ❯ sway -v
> swayfx version 0.4-5ee4d737 (Oct 30 2024, branch 'master') (based on sway 1.9.0)
> ```
> 
> I'm having the same issue.
> 
> @hrqmonteiro Ever figured it out?

Stopped trying to use Sway :)

Moved to Hyprland and it all worked, also if i have some issues, people ACTUALLY give me a solution, the total opposite of what they did here lol

**Comment by DovieW** [comment]
i gave up too lol. im back to i3 and have my eye on hyprland

**Comment by ErikReider** [comment]
> ```
> 
> ❯ sway -v
> 
> swayfx version 0.4-5ee4d737 (Oct 30 2024, branch 'master') (based on sway 1.9.0)
> 
> ```
> 
> 
> 
> I'm having the same issue.
> 
> 
> 
> @hrqmonteiro Ever figured it out?

So sway --version outputs the correct version, what if you just run the command "sway" manually inside a tty or another compositor?

**Comment by ErikReider** [comment]
> Moved to Hyprland and it all worked, also if i have some issues, people ACTUALLY give me a solution, the total opposite of what they did here lol

You do know that both me and @WillPower3309 work on this + other projects on our spare time, for free after working/studying right?

**Comment by DovieW** [comment]
> So sway --version outputs the correct version, what if you just run the command "sway" manually inside a tty or another compositor?

So that's how I was running it at the time, from a tty. In case it matters, I'm using the `--unsupported-gpu` flag. I just tried again now, and it actually did work this time. So nevermind and thank you. Maybe I needed a reboot or something.
